setup.py

```diff
@@ -6,314 +6,5 @@
 # License: 3-clause BSD.  The full license text is available at:
 #  - https://github.com/uqfoundation/dill/blob/master/LICENSE
 
-import os
-
-# set version numbers
```

Comment:
Not sure I like the change in version numbering.  I need to ponder that a bit...